packages/clerk-js/src/ui/utils/useFormControl.ts

```diff
@@ -177,82 +166,26 @@ export const buildRequest = (fieldStates: Array<FormControlStateLike>): Record<s
 
 type DebouncedFeedback = {
   debounced: {
-    errorText: string;
-    warningText: string;
-    successfulText: string;
-    isFocused: boolean;
-    informationText: string | LocalizationKey;
+    feedback: string;
+    feedbackType: FeedbackType;
   };
 };
 
 type DebouncingOption = {
-  hasLostFocus: boolean;
-  warningText: string | undefined;
-  errorText: string | undefined;
-  enableErrorAfterBlur: boolean | undefined;
-  successfulText: string | undefined;
-  isFocused: boolean;
-  informationText: LocalizationKey | string | undefined;
-  hasPassedComplexity: boolean;
-  skipBlur?: boolean;
+  feedback?: string;
+  feedbackType?: FeedbackType;
+  isFocused?: boolean;
```

Comment:
This seems to be removed from useFormControl, are we now passing it manually ? Do we have any benefit from this ?

Reply:
After a huddle, we came to the conclusion that the issue this solved, is probably not an issue anymore.